userspace/libsinsp/parsers.cpp

```diff
@@ -4583,15 +4623,26 @@ void sinsp_parser::parse_container_json_evt(sinsp_evt *evt)
 
 		const Json::Value &port_mappings = container["port_mappings"];
 
-		if(!port_mappings.isNull() && port_mappings.isConvertibleTo(Json::arrayValue))
+		if(!port_mappings.isNull() && check_is_convertible(port_mappings,Json::arrayValue,std::string("port_mappings")))
 		{
 			for (Json::Value::ArrayIndex i = 0; i != port_mappings.size(); i++)
 			{
 				sinsp_container_info::container_port_mapping map;
-				map.m_host_ip = port_mappings[i]["HostIp"].asInt();
-				map.m_host_port = (uint16_t) port_mappings[i]["HostPort"].asInt();
-				map.m_container_port = (uint16_t) port_mappings[i]["ContainerPort"].asInt();
-
+				const Json::Value &host_ip = port_mappings[i]["HostIp"];
+				// We log message for HostIp conversion failure at Warning level
```

Comment:
We ran into an issue on the field where a customer got segfault for these particular fields. It would be good to receive some info about these for now; we can always switch the flags later. 

Why we don't do this for all fields is : for example there is a field called `SwapMemoryLimit` or some such. It is supposed to be a `+ve` number but it also allows `-1` for `unlimited` case. So weird cases like that will fill the logs. So we want it at debug for most. For just these parameters of interest; we switch to warning.